### DISCUSSION TITLE
Planning maxilla/mandible from hip reconstruction surgery and various others questions

### DISCUSSION THREAD

**Discussion by pan-daniel** [lassoan/SlicerBoneReconstructionPlanner#116] [Q&A]
Hello! 

I am writing on behalf of my friend, who uses your software extensively to plan his surgeries (he doesn't have a GitHub account) and he is incredibly happy with it when it comes to the fibula reconstruction. He has first seen your software through some YouTube video, but since then I tracked down you here on GitHub and I hope you do not mind some questions from him.

1. Did you see any use of your plug-in when it comes to planning and creating maxilla/mandible from hip reconstruction surgery? Maybe there exists some tutorial for it or you have pointers what could be useful in achieving that?

Currently, he has the following idea (and please excuse me if I am being unclear, I am trying to move in the terminology that is mostly unknown to me personally. Feel free to ask for any clarification or screenshots): 

- He marks the cutting planes on the jaw and there he makes a cutting guide
- Then, he makes a shell of the jaw part with the osteotomy axis preserved
- He changes the program to Meshmixer and tries to transfer to the hip, adapting the shell shape to the shape of the hip disc

2. Is there a possibility for the slot on the saw to widen symmetrically with respect to the cutting plane and not just outward because he wants to put the metal bushings there in the middle in the template?
3. Perhaps you do know of community that talks about similar topics?

Greetings from Switzerland and Poland
Daniel and Hubert

**Comment by mauigna06** [comment]
Hi Daniel and Hubert

Thanks for writing. Please give me some time to answer :)

**Reply by pan-daniel** [reply]
Of course, no problem : )

**Comment by cmfsx** [comment]
Hi  Daniel and Hubert

If you could share some photos of this workflow i think we might be able to suggest. Basically if you are looking at DCIA it can be worked out within Slicer albeit not straightforward. 

best

**Reply by pan-daniel** [reply]
Please give us time to procure it. Thank you. :)

**Reply by pan-daniel** [reply]
First, in 3D Slicer, convert the DICOM files of the maxilla and hip bone to STL format. Next, delineate the resection area, excise that element, and thicken its surface to create a shell, which will serve as the template base (highlighted in red in the image).

![obraz](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/59827851/7f6b6f1b-37ea-4202-923f-5d0e266b5879)
![obraz](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/59827851/91b268f8-517a-4d3f-8e6c-90b59cd8e350)

Proceed to Meshmixer for manually setting up cutting guides and connecting them to the template base.
![obraz](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/59827851/e6d079ba-1baf-4387-b794-cf0d844f55ca)
![obraz](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/59827851/4e717f12-d04e-42ab-87f4-f7a63cd2baa2)

Utilize Boolean difference to ensure the guides are tailored to the shape of the maxilla, resulting in a ready-to-use resection template. Subsequently, cut the resection template base and transfer the guides to the hip, extending them and adding beams to connect the guides and serve as an anchor to the hip.
![obraz](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/59827851/48ffd911-e685-49da-8ac3-bd66c6216d48)

Apply Boolean difference to fit the template to the hip surface. While the process may sound straightforward, in his opinion it lacks precision.

An alternative approach is to use the red element on the maxilla as a template without adding guides for cutting. The saw will then follow its edges. Transfer this red element to the hip, creating a shape with straight sides corresponding to the resected element, serving as a reconstructive template.

That's it. Hope it is understandable. : ) Thank you for help.

**Reply by mauigna06** [reply]
Thank you for taking time to explain the workflow. I do not see why it should not be possible to translate it completely to Slicer. Hopefully could be done this year. I may have help from some student
